test/test_cases/TEST_ID_16_providing_complex_secret_successfully.sh

```diff
@@ -0,0 +1,24 @@
+#!/bin/bash
+set -euxo pipefail
+
+create_secret_access_role
+
+create_secret_access_role_binding
+
+set_namespace "$CONJUR_NAMESPACE_NAME"
+conjur_cli_pod=$(get_conjur_cli_pod_name)
+secret_value="{\"auths\":{\"someurl\":{\"auth\":\"sometoken=\"}}}"
+$cli_with_timeout "exec $conjur_cli_pod -- conjur variable values add secrets/test_secret $secret_value"
+set_namespace "$TEST_APP_NAMESPACE_NAME"
+
+deploy_test_env
+
+echo "Verifying pod test_env has environment variable 'TEST_SECRET' with value '$secret_value'"
+pod_name=$(cli_get_pods_test_env | awk '{print $1}')
+verify_secret_value_in_pod "$pod_name" TEST_SECRET "$secret_value"
+
+# set the secret back to original one for the rest of the tests
+set_namespace "$CONJUR_NAMESPACE_NAME"
```

Comment:
I am not sure we need this last part because for every test we run 
```
  for filename in ./$TEST_NAME_PREFIX*.sh; do
      announce "Running '$filename'."
      ./test_case_setup.sh
      $filename
      ./test_case_teardown.sh
      announce "Test '$filename' ended successfully"
  done
```
so the teardown is happening after each test execution and looking at `TEST_ID_1`, it looks similar to this test and this isn't performed so it might not be needed. What do you think?

Reply:
You are correct, thanks!